tests/test_lab_model_convert.py

```diff
@@ -1,98 +0,0 @@
-# Standard
```

Comment:
Any reason all of the convert tests are being removed here?

Reply:
i needed to rewrite them, will add in a follow up commit